### PR TITLE
Replace custom screen blank code with Nemo.KeepAlive

### PR DESCRIPTION
**Pull request by mlehtima**
Nemo.KeepAlive 1.2 is now allowed in Jolla Store.

### CODE REVIEW

File: harbour-nayttamo.pro [hunk 1]
```diff
@@ -17,14 +17,29 @@ CONFIG += sailfishapp libcrypto
 SOURCES += src/harbour-nayttamo.cpp \
     src/urldecrypt.cpp
 
-OTHER_FILES += \
-    qml/cover/CoverPage.qml \
-    qml/pages/FirstPage.qml \
+DISTFILES += \
```
Comment (joonne) [inline harbour-nayttamo.pro:20]: This seems nice but what's the reasoning with `DISTFILES` vs `OTHER_FILES` ?

File: qml/pages/PlayerPage.qml [hunk 2]
```diff
@@ -53,8 +54,8 @@ Page {
     // The effective value will be restricted by ApplicationWindow.allowedOrientations
     allowedOrientations: Orientation.All
 
-    ScreenBlank {
-        enabled: mediaPlayer.playbackState === MediaPlayer.PlayingState
+    DisplayBlanking {
```
Comment (joonne) [inline qml/pages/PlayerPage.qml:57]: Cool :clap: :+1:

File: qml/pages/SearchPage.qml [hunk 3]
```diff
@@ -101,5 +96,10 @@ Page {
         VerticalScrollDecorator {
             id: decorator
         }
+
+        ViewPlaceholder {
```
Comment (joonne) [inline qml/pages/SearchPage.qml:100]: :+1:

**Review by joonne — APPROVED** [review]

File: harbour-nayttamo.pro (hunk 1 above)
Comment (mlehtima) [reply harbour-nayttamo.pro:20]: Those both work but having files split into two variables is confusing and I selected the DISTFILES based on what was being used on some other apps.

**Pull request merged**